service/app/controllers/LearnerController.java

```diff
@@ -29,16 +29,14 @@
 public class LearnerController extends BaseController {
 
     private LearnerStateRequestValidator validator = new LearnerStateRequestValidator();
-    private ActorRef learnerStateUpdateActorRef;
-    private ActorRef learnerStateActorRef;
 
+    @Inject
+    @Named("learner-state-update-actor")
+    private ActorRef learnerStateUpdateActorRef;
 
     @Inject
-    public LearnerController(@Named("learner-state-update-actor") ActorRef learnerStateUpdateActorRef,
-                                      @Named("learner-state-actor") ActorRef learnerStateActorRef) {
-        this.learnerStateUpdateActorRef = learnerStateUpdateActorRef;
-        this.learnerStateActorRef = learnerStateActorRef;
-    }
+    @Named("learner-state-actor")
```

Comment:
please get rid of all the blank lines between the annotation and variable declaration